debian/control-template

```diff
@@ -5,6 +5,13 @@ Build-Depends: debhelper (>= 9),
  cmake (>= 2.8),
  graphviz,
  python (>= 2.7),
+ python-sphinx,
+ python-sphinx-rtd-theme | base-files (<= 7.2ubuntu5.6~),
+ graphviz,
+ texlive-fonts-recommended,
```

Comment:
Are those dependencies meant for the docs package or only for the generation of the docs package?

Reply:
these dependencies are Build-Dependencies, they are not needed or requested if one installs the docs